src/Parser/DisciplinePriest/Modules/Features/AtonementSource.js

```diff
@@ -0,0 +1,44 @@
+import SPELLS from 'common/SPELLS';
+
+import Module from 'Parser/Core/Module';
+
+class AtonementSource extends Module {
+  _previousDamageEvent = null;
+  _previousAtonementApplicator = null;
+
+  // Spells that will apply atonement
+  atonementApplicators = new Map([
```

Comment:
Why are you using a `Map`? What does a `Map` have that an object doesn't?

I think this would essentially be the same:

```
atonementApplicators = {
  [SPELLS.POWER_WORD_RADIANCE.id]: SPELLS.POWER_WORD_RADIANCE.atonementDuration,
  [SPELLS.POWER_WORD_SHIELD.id]: SPELLS.POWER_WORD_SHIELD.atonementDuration,
  [SPELLS.PLEA.id]: SPELLS.PLEA.atonementDuration,
  [SPELLS.SHADOW_MEND.id]: SPELLS.SHADOW_MEND.atonementDuration,
};
```